### DISCUSSION TITLE
쿠키와 세션의 차이점이 뭘까요?

### DISCUSSION THREAD

**Discussion by Hyorim-Kim** [Hyorim-Kim/Team#60] [Q&A] [answered]
개념 설명해주세요

**Comment by Youkwangjin** [comment, marked as answer]
일반적으로 웹 애플리케이션은 쿠키와 세션을 조합하여 사용자 경험을 향상시키고 보안을 유지하는 데 활용합니다. 예를 들어, 사용자 식별 정보는 세션에 저장하고, 사용자 환경 설정과 같은 덜 중요한 데이터는 쿠키를 통해 유지할 수 있습니다.
